### PR TITLE
Added `Modal` for Login Multiaccounts select

### PR DESCRIPTION
**Pull request by ThePhisch**
Within the login UI, users can select an account, or have one mapped to them. If multiple accounts can be mapped to the given credentials, the new modal will show up, prompting the user to select an account.

To facilitate this, added the optional `rucioMultiAccount?: string` to `AuthViewModel`, this is a comma-separated string of the possible account names.

Uses `react-modal` addon. In order to be able to style the modal and the overlay (overlay ~= the container into which the modal is placed, ie the container it is hovering in) using tailwind, the regex to discover tailwind classes was adapted.

### CODE REVIEW

**Comment by maany** [comment]
Please rebase to see if the `react-modal` installation works with NodeJS 18 and NodeJS 20 instead :)

File: test/component/Login.test.tsx [hunk 1]
```diff
@@ -190,10 +192,12 @@ describe("Login Page Test", () => {
...
             rucioAccount: "",
+            rucioMultiAccount: "",
             rucioAuthType: "",
             rucioAuthToken: "",
             rucioIdentity: "",
             rucioAuthTokenExpires: "",
+            role: undefined,
         }
 
         await act( async () => render(<LoginStory
```
Comment (maany) [inline test/component/Login.test.tsx:203]: Can we add a test to ensure that the modal shows up when `rucioMultiAccount` is not empty :)

Comment (ThePhisch) [reply test/component/Login.test.tsx:246]: done
